core/deployment/src/main/java/io/quarkus/deployment/dev/testing/TestConfig.java

```diff
@@ -15,6 +15,7 @@
  * <p>
  * TODO refactor code to actually use these values
  */
+@SuppressWarnings("jol")
```

Comment:
Never seen this. What is it for?

Reply:
Huh, I didn't actually add it... Maybe the IDE did... Let me remove it